Configuration/NodeTypes.yaml

```diff
@@ -177,3 +177,12 @@
     'text':
       search:
         fulltextExtractor: '${Indexing.extractHtmlTags(value)}'
+
+'TYPO3.Neos:Document':
+  properties:
+    'title':
+      search:
+        elasticSearchMapping:
+          type: string
+          include_in_all: true
+          boost: 20
```

Comment:
this will override the existing configuration for `TYPO3.Neos:Document`, it needs to be moved up to the first definition instead